README.md

```diff
@@ -71,6 +70,7 @@ repositories {
 
 dependencies {
     testImplementation 'org.mockbukkit.mockbukkit:mockbukkit-v1.21:[version]'
+    testImplementation 'io.papermc.paper:paper-api:[version]'
```

Comment:
maybe make this a variable? so people only need to change their variable section?